### ISSUE TITLE
Disabled extension text don't meet needed contrast levels

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#67433]
In Dark+ this is #767676 on #252526 which fails WCAG AA for normal sized text

Forked from https://github.com/Microsoft/vscode/issues/66931

**Comment by misolori** [comment]
I believe disabled text falls under the "incidental" category, which means it's an exception:

https://webaim.org/articles/contrast/

<img width="643" alt="image" src="https://user-images.githubusercontent.com/35271042/52091229-9188d100-2568-11e9-92c0-c6c51a28b6cc.png">
